don't use bundled libraries for libpng, zlib, freetype, and utf8proc,
use system libraries instead

--- a/linux/Makefile.am	2025-12-26 15:32:32.000000000 +0100
+++ b/linux/Makefile.am	2025-12-27 10:11:07.069452562 +0100
@@ -32,28 +32,6 @@
 				/usr/include/gpac/internal/odf_dev.h \
 				/usr/include/gpac/internal/odf_parse_common.h \
 				/usr/include/gpac/internal/ogg.h \
-				../src/thirdparty/libpng/pngstruct.h \
-				../src/thirdparty/libpng/pngpriv.h \
-				../src/thirdparty/libpng/pnginfo.h \
-				../src/thirdparty/libpng/pnglibconf.h \
-				../src/thirdparty/libpng/pngconf.h \
-				../src/thirdparty/libpng/pngdebug.h \
-				../src/thirdparty/libpng/png.h \
-				../src/thirdparty/libpng/png.c \
-				../src/thirdparty/libpng/pngerror.c \
-				../src/thirdparty/libpng/pngget.c \
-				../src/thirdparty/libpng/pngmem.c \
-				../src/thirdparty/libpng/pngpread.c \
-				../src/thirdparty/libpng/pngread.c \
-				../src/thirdparty/libpng/pngrio.c \
-				../src/thirdparty/libpng/pngrtran.c \
-				../src/thirdparty/libpng/pngrutil.c \
-				../src/thirdparty/libpng/pngset.c \
-				../src/thirdparty/libpng/pngtrans.c \
-				../src/thirdparty/libpng/pngwio.c \
-				../src/thirdparty/libpng/pngwrite.c \
-				../src/thirdparty/libpng/pngwtran.c \
-				../src/thirdparty/libpng/pngwutil.c \
 				../src/lib_ccx/ccx_common_common.h \
 				../src/lib_ccx/ccx_common_option.h \
 				../src/lib_ccx/utility.h \
@@ -172,34 +150,6 @@
 				../src/lib_ccx/ts_tables_epg.c \
 				../src/lib_ccx/wtv_constants.h \
 				../src/lib_ccx/wtv_functions.c \
-				../src/thirdparty/zlib/adler32.c \
-				../src/thirdparty/zlib/compress.c \
-				../src/thirdparty/zlib/crc32.c \
-				../src/thirdparty/zlib/crc32.h \
-				../src/thirdparty/zlib/deflate.c \
-				../src/thirdparty/zlib/deflate.h \
-				../src/thirdparty/zlib/gzclose.c \
-				../src/thirdparty/zlib/gzguts.h \
-				../src/thirdparty/zlib/gzlib.c \
-				../src/thirdparty/zlib/gzread.c \
-				../src/thirdparty/zlib/gzwrite.c \
-				../src/thirdparty/zlib/infback.c \
-				../src/thirdparty/zlib/inffast.c \
-				../src/thirdparty/zlib/inffast.h \
-				../src/thirdparty/zlib/inffixed.h \
-				../src/thirdparty/zlib/inflate.c \
-				../src/thirdparty/zlib/inflate.h \
-				../src/thirdparty/zlib/inftrees.c \
-				../src/thirdparty/zlib/inftrees.h \
-				../src/thirdparty/zlib/trees.c \
-				../src/thirdparty/zlib/trees.h \
-				../src/thirdparty/zlib/uncompr.c \
-				../src/thirdparty/zlib/zconf.h \
-				../src/thirdparty/zlib/zlib.h \
-				../src/thirdparty/zlib/zutil.c \
-				../src/thirdparty/zlib/zutil.h \
-				../src/thirdparty/utf8proc/utf8proc.c \
-				../src/thirdparty/utf8proc/utf8proc.h \
 				../src/thirdparty/lib_hash/sha2.c \
 				../src/thirdparty/lib_hash/sha2.h \
 				../src/lib_ccx/zvbi/bcd.h \
@@ -213,67 +163,18 @@
 				../src/lib_ccx/zvbi/sampling_par.c \
 				../src/lib_ccx/zvbi/sampling_par.h \
 				../src/lib_ccx/zvbi/sliced.h \
-				../src/lib_ccx/zvbi/zvbi_decoder.h \
-				../src/freetype/* \
-				../src/thirdparty/freetype/autofit/autofit.c \
-				../src/thirdparty/freetype/base/ftbase.c \
-				../src/thirdparty/freetype/base/ftbbox.c \
-				../src/thirdparty/freetype/base/ftbdf.c \
-				../src/thirdparty/freetype/base/ftbitmap.c \
-				../src/thirdparty/freetype/base/ftcid.c \
-				../src/thirdparty/freetype/base/ftfntfmt.c \
-				../src/thirdparty/freetype/base/ftfstype.c \
-				../src/thirdparty/freetype/base/ftgasp.c \
-				../src/thirdparty/freetype/base/ftglyph.c \
-				../src/thirdparty/freetype/base/ftgxval.c \
-				../src/thirdparty/freetype/base/ftinit.c \
-				../src/thirdparty/freetype/base/ftlcdfil.c \
-				../src/thirdparty/freetype/base/ftmm.c \
-				../src/thirdparty/freetype/base/ftotval.c \
-				../src/thirdparty/freetype/base/ftpatent.c \
-				../src/thirdparty/freetype/base/ftpfr.c \
-				../src/thirdparty/freetype/base/ftstroke.c \
-				../src/thirdparty/freetype/base/ftsynth.c \
-				../src/thirdparty/freetype/base/ftsystem.c \
-				../src/thirdparty/freetype/base/fttype1.c \
-				../src/thirdparty/freetype/base/ftwinfnt.c \
-				../src/thirdparty/freetype/bdf/bdf.c \
-				../src/thirdparty/freetype/bzip2/ftbzip2.c \
-				../src/thirdparty/freetype/cache/ftcache.c \
-				../src/thirdparty/freetype/cff/cff.c \
-				../src/thirdparty/freetype/cid/type1cid.c \
-				../src/thirdparty/freetype/gzip/ftgzip.c \
-				../src/thirdparty/freetype/include/ft2build.h \
-				../src/thirdparty/freetype/lzw/ftlzw.c \
-				../src/thirdparty/freetype/pcf/pcf.c \
-				../src/thirdparty/freetype/pfr/pfr.c \
-				../src/thirdparty/freetype/psaux/psaux.c \
-				../src/thirdparty/freetype/pshinter/pshinter.c \
-				../src/thirdparty/freetype/psnames/psnames.c \
-				../src/thirdparty/freetype/raster/raster.c \
-				../src/thirdparty/freetype/sfnt/sfnt.c \
-				../src/thirdparty/freetype/smooth/smooth.c \
-				../src/thirdparty/freetype/truetype/truetype.c \
-				../src/thirdparty/freetype/type1/type1.c \
-				../src/thirdparty/freetype/type42/type42.c \
-				../src/thirdparty/freetype/winfonts/winfnt.c
-
-if SYS_IS_APPLE_SILICON
-ccextractor_SOURCES += 	../src/thirdparty/libpng/arm/arm_init.c \
-  						../src/thirdparty/libpng/arm/filter_neon_intrinsics.c \
-  						../src/thirdparty/libpng/arm/palette_neon_intrinsics.c
-endif
+				../src/lib_ccx/zvbi/zvbi_decoder.h
 
-ccextractor_CFLAGS = -std=gnu99 -Wno-write-strings -Wno-pointer-sign -D_FILE_OFFSET_BITS=64 -DVERSION_FILE_PRESENT -DFT2_BUILD_LIBRARY -DGPAC_DISABLE_VTT -DGPAC_DISABLE_OD_DUMP -DGPAC_DISABLE_REMOTERY -DNO_GZIP
+ccextractor_CFLAGS = -std=gnu99 -Wno-write-strings -Wno-pointer-sign -D_FILE_OFFSET_BITS=64 -DVERSION_FILE_PRESENT -DGPAC_DISABLE_VTT -DGPAC_DISABLE_OD_DUMP -DGPAC_DISABLE_REMOTERY -DNO_GZIP
 
-ccextractor_CPPFLAGS =-I../src/lib_ccx/ -I/usr/include/ -I../src/thirdparty/libpng/ -I../src/thirdparty/zlib/ -I../src/lib_ccx/zvbi/ -I../src/thirdparty/lib_hash/ -I../src/thirdparty -I../src/ -I../src/thirdparty/freetype/include/
+ccextractor_CPPFLAGS =-I../src/lib_ccx/ -I@@XBPS_CROSS_BASE@/usr/include/ -I../src/lib_ccx/zvbi/ -I../src/thirdparty/lib_hash/ -I../src/thirdparty -I../src/ -I@@XBPS_CROSS_BASE@@/usr/include/freetype2/
 
 
-ccextractor_LDADD=-lm -lpthread -ldl -lgpac
+ccextractor_LDADD=-lm -lpthread -ldl -lgpac -lz -lpng -lfreetype -lutf8proc
 
 
 if SYS_IS_LINUX
-ccextractor_CFLAGS += -O3 -s
+ccextractor_CFLAGS += -O2
 endif
 
 if SYS_IS_MAC
